File: 2013/UnSystem/Applications/DBAccess/Define.vb
﻿
' 項目値定義一覧(※固定値を管理)
Public Class Define

    ' 操作レベル
    Enum OpeLevel
        PawarUser = 1
        Negotiation = 2
        Personal = 3
        Administrator = 9
    End Enum

    ' 就業状況
    Enum WorkStatus
        None
        Free
        Work
    End Enum

    ' 閾値
    Enum ThresholdStatus
        AndOver
        AndLess
        Equal
    End Enum

    ' 性別
    Enum Sex
        None
        Man
        Woman
    End Enum

    ' メッセージ(※メッセージは全てここに定義)
    Enum Message
        SystemErr
        Login_InputErr
        Notfound
    End Enum

End Class

File: 2014/勤怠アプリ学習/WebApplication2/WebApplication2/DBAccess/Define.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApplication2.DBAccess
{
    public class Define
    {
        //認証レベル
        enum OpeLevel
        {
            PawarUser = 1,
            Negotiation = 2,
            Personal = 3,
            Administrator = 9
        };

        //システムメッセージ
        enum Message
        {
            SystemErr,
            Login_InputErr,
            Notfound
        };
    }
}